packages/components/divider/src/divider.vue

```diff
@@ -9,30 +9,20 @@
   </div>
 </template>
 
-<script lang="ts">
-import { defineComponent, computed } from 'vue'
+<script lang="ts" setup>
+import { computed, defineProps } from 'vue'
```

Comment:
**Nit:** we don't really need to import this method `defineProps` because it was globally added via marcos

Reply:
~ get it